.github/workflows/main.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   push:
-    branches: [main, opensuse]
+    branches: [main, opensuse, GSoC23]
```

Comment:
Let's remove this from the workflow, because right now there's a PR so this branch should be tested by the `pull_request` action.